src/types/ChangemakerRole.ts

```diff
@@ -0,0 +1,36 @@
+import { ajv } from '../ajv';
+import { AccessType } from './AccessType';
+import type { JSONSchemaType } from 'ajv';
+import type { Writable } from './Writable';
+import type { KeycloakUserId } from './KeycloakUserId';
+
+interface ChangemakerRole {
+	readonly changemakerId: number;
+	readonly userKeycloakUserId: KeycloakUserId;
+	readonly accessType: AccessType;
```

Comment:
2. At first glance a column seemed to be missing. I would expect "subject permission object" to be declared somewhere. For example, "Alice can manage proposals for changemaker Bob." Where does the object such as "changemaker Bob" fit in here? Oh, is it `changemakerId`? Perhaps the order is confusing or there is some ambiguity. I thought that this `Role` declared "User `userKeycloakUserId` is part of Changemaker `changemakerId`" (`changemakerId` is part of the subject) rather than "User `userKeycloakUserId` has permission `accessType` on object `changemakerId` (`changemakerId` is part of the object). It is still unclear as of writing this but perhaps it will be clear while looking further.

Reply:
I'll describe the intention here and we can work from there to land on where to go:

The intention is that a record in this table would say "`userKeycloakUserUd` has permission to `accessType` data related to `ChangemakerId`.

For instance: `{ Alice } has permission to { edit proposal data } related to { Foo Inc. }`

Reply:
The names AND attribute order are now all updated to hopefully make more intuitive sense.